src/jdk.jlink/share/classes/jdk/tools/jlink/internal/ImageResourcesTree.java

```diff
@@ -300,7 +300,7 @@ public String toPackageName(Node node) {
                 return null;
             }
             String pkg = removeRadical(path, module + "/");
-            return pkg.replaceAll("/", ".");
+            return pkg.replace("/", ".");
```

Comment:
With the conversion to `replace`, we can use the char literals `'/'` and `'.'` instead of the string literals, which gives better performance as well.